### PR TITLE
fix: configurable products button

### PR DESCRIPTION
**Pull request by ivan-kalachikov**
## Description

image
image
image
image

### Jira-link:
https://virtocommerce.atlassian.net/browse/VCST-2356
### Artifact URL:
https://vc3prerelease.blob.core.windows.net/packages/vc-theme-b2b-vue-2.11.0-pr-1484-7e9a-7e9afc20.zip

### CODE REVIEW

**Review by Andrew-Orlov — APPROVED** [review]

File: client-app/shared/catalog/components/product-card-compare.vue [hunk 1]
```diff
@@ -28,19 +28,29 @@
...
+      <template v-if="product.isConfigurable || product.hasVariations">
+        <VcProductButton
+          v-if="product.isConfigurable"
+          class="mb-4"
+          :to="link"
+          :button-text="$t('pages.catalog.customize_button')"
+          icon="cube-transparent"
+          :target="$cfg.details_browser_target"
+        />
+        <VcButton
```
Comment (goldenmaya) [inline client-app/shared/catalog/components/product-card-compare.vue:40]: use VcProductButton here too

File: client-app/shared/catalog/components/product-card-related.vue [hunk 2]
```diff
@@ -31,18 +31,28 @@
...
+      <template v-if="product.isConfigurable || product.hasVariations">
+        <VcProductButton
+          v-if="product.isConfigurable"
+          :to="link"
+          :button-text="$t('pages.catalog.customize_button')"
+          icon="cube-transparent"
+          target="_blank"
+        />
+
+        <VcButton
```
Comment (goldenmaya) [inline client-app/shared/catalog/components/product-card-related.vue:43]: use VcProductButton here too

File: client-app/shared/wishlists/components/wishlist-line-items.vue [hunk 3]
```diff
@@ -33,9 +33,30 @@
...
+          <template v-if="item.isConfigurable || item.hasVariations">
+            <VcProductButton
+              v-if="item.isConfigurable"
+              no-wrap
+              :to="item.route"
+              :button-text="$t('pages.catalog.customize_button')"
+              icon="cube-transparent"
+              :target="$cfg.details_browser_target"
+            />
+            <VcButton
```
Comment (goldenmaya) [inline client-app/shared/wishlists/components/wishlist-line-items.vue:46]: use VcProductButton here too

File: client-app/shared/catalog/components/product-card-compare.vue [hunk 4]
```diff
@@ -28,19 +28,29 @@
...
+          v-if="product.hasVariations"
```
Comment (goldenmaya) [inline client-app/shared/catalog/components/product-card-compare.vue:41]: v-else-if ?

File: client-app/shared/catalog/components/product-card-recently-browsed.vue [hunk 5]
```diff
@@ -13,11 +13,22 @@
...
+      <VcProductButton
+        v-if="product.isConfigurable"
+        :to="link"
+        :button-text="$t('pages.catalog.customize_button')"
+        icon="cube-transparent"
+        target="_blank"
+      />
+
+      <VcProductButton
+        v-if="product.hasVariations"
```
Comment (goldenmaya) [inline client-app/shared/catalog/components/product-card-recently-browsed.vue:26]: v-else-if ?

**Pull request merged**